dev/docker-compose.yml

```diff
@@ -145,5 +147,13 @@ services:
     entrypoint: sleep
     command: infinity
 
+  jwks:
+    image: nginx
+    ports:
+      - 8880:80
+    volumes:
+      - jwks-volume:/usr/share/nginx/html
+
 volumes:
```

Comment:
Why empty volumes here?

Reply:
What does it mean empty?
The volume will be populated during runtime.
It's actually a way jwks-cucumber containers "communicate" each other